.travis.yml

```diff
@@ -1,116 +0,0 @@
-os: linux
```

Comment:
I think it's finally time to remove this file instead of trying to keep it up to date.